config/fund_loader_config/cof/cof_r4.py

```diff
@@ -253,19 +253,28 @@
         "contact_textphone": None,
         "support_times": "9am to 5pm",
         "support_days": "Monday to Friday",
-        "instructions": (
-            "You must have received an invitation to apply. If we did not invite you,"
-            " first <a"
-            ' href="https://www.gov.uk/guidance/community-ownership-fund-round-3-how-'
-            'to-express-your-interest-in-applying">'
-            " express your interest in the fund</a>."
-        ),
+        "instructions_json": {
+            "en": (
+                "You must have received an invitation to apply. If we did not invite you,"
+                " first <a"
+                ' href="https://www.gov.uk/guidance/community-ownership-fund-round-3-how-'
+                'to-express-your-interest-in-applying">'
+                " express your interest in the fund</a>."
+            ),
+            "cy": (
```

Comment:
Have you got a ticket to get this translated?

Reply:
Translations done